arduino/sensors/Adafruit-AM2315/Kconfig

```
@@ -0,0 +1,31 @@
+
+# Kconfig file for package Adafruit-AM2315
+menuconfig PKG_USING_ARDUINO_ADAFRUIT_AM2315
+    bool "Adafruit AM2315 Humidity & Temp sensor"
```

这里搞错了 Kconfig需要冒号 json里不需要冒号
```suggestion
    bool "Adafruit AM2315: Humidity & Temp sensor"
```